packages/replay-next/src/suspense/FocusIntervalCache.ts

```diff
@@ -51,7 +53,18 @@ export function createFocusIntervalCache<
     ...options,
     load: async (start, end, ...paramsWithCacheLoadOptions) => {
       try {
-        return await options.load(start, end, ...paramsWithCacheLoadOptions);
+
+        const startTime = Date.now();
+        const response = await options.load(start, end, ...paramsWithCacheLoadOptions);
+        const stopTime = Date.now();
+        if (options.telemetry) {
+          recordTelemetryData("suspense-cache-load", {
+            duration: stopTime - startTime,
+            label: options.debugLabel,
+            params: { start, end, ...paramsWithCacheLoadOptions },
+          });
+        }
```

Comment:
This would be a very noisy metric for us to collect, as this cache runs *a lot*. E